### DISCUSSION TITLE
SetAccelerator on V3 whats wrong?

### DISCUSSION THREAD

**Discussion by adarosci** [wailsapp/wails#3629] [General]
I'm implement a basic app for my context and I trying to add a accelerator on Menu but apparently does not work or I doing anything wrong? 

![image](https://github.com/user-attachments/assets/f9f22f20-fe22-48f2-affc-921d4d41299e)

I'm set menu before the run

![image](https://github.com/user-attachments/assets/dda4e342-5e84-4268-bced-10b2bcca2e1e)

Apparently the accelerator dont set becouse dont show on the menu

![image](https://github.com/user-attachments/assets/3ba9d17e-a557-4de8-be33-a0a05261f6ab)

I don't know if this has already been implemented

**Comment by leaanthony** [comment]
You'll get better help on discord 👍
